src/azurelinux/3.0/net9.0/cross/riscv64/Dockerfile

```diff
@@ -5,11 +5,11 @@ ARG ROOTFS_DIR
 
 # Install the latest debootstrap
 RUN tdnf remove -y debootstrap && \
```

Comment:
```suggestion
# switch to https://deb.debian.org/debian/pool/main/d/debootstrap/debootstrap_1.0.137.tar.gz when it's released
RUN tdnf remove -y debootstrap && \
```

Reply:
This should be filed as a separate issue.

Reply:
Yep, I plan to once this is merged.